docs/root/configuration/http/http_filters/wasm_filter.rst

```diff
@@ -21,20 +21,12 @@ Example configuration
 
 Example filter configuration:
 
-.. code-block:: yaml
-
-  name: envoy.filters.http.wasm
-  typed_config:
-    "@type": type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
-    config:
-      config:
-        name: "my_plugin"
-        vm_config:
-          runtime: "envoy.wasm.runtime.v8"
-          code:
-            local:
-              filename: "/etc/envoy_filter_http_wasm_example.wasm"
-          allow_precompiled: true
+.. literalinclude:: ../../../start/sandboxes/_include/wasm-cc/envoy.yaml
+    :language: yaml
+    :lines: 24-49
+    :emphasize-lines: 4-21
+    :linenos:
 
+   :caption: :download:`wasm envoy.yaml <../../../start/sandboxes/_include/wasm-cc/envoy.yaml>`
```

Comment:
It looks that you've accidentally(?) moved this out of the `literalinclude` block.

Reply:
yep -  typo - fixed